navigation/navigation-runtime/src/jsTest/kotlin/androidx/navigation/BrowserHistoryTest.kt

```diff
@@ -50,7 +50,7 @@ class BrowserHistoryTest {
 
     @Test
     fun checkBrowserHistoryStateSynchronizedWithNavigation() = runTest {
-        cleanWindowHistory()
+        val initHistoryLength = goToBrowserRoot()
```

Comment:
Could you please add the test task of this project as a dependency in `:mpp:webTest` ?